docs: enable code coverage. (#5928)
codeigniter4/CodeIgniter4: tests/README.md

	> ln -s ./vendor/bin/phpunit ./phpunit

You also need to install [XDebug](https://xdebug.org/docs/install) in order

> for code coverage to be calculated successfully. After installation `XDebug`. You must add ``xdebug.mode=coverage`` in file **php.ini**. Because code coverage needs to be enable.

Reviewer: ```suggestion
for code coverage to be calculated successfully. After installing `XDebug`, you must add `xdebug.mode=coverage` in the **php.ini** file to enable code coverage.
```

> for code coverage to be calculated successfully. After installation `XDebug`. You must add ``xdebug.mode=coverage`` in file **php.ini**. Because code coverage needs to be enable.Then do not forget to restart **Apache**.

Reviewer 2: I believe Apache has nothing to do with. `phpunit` runs on CLI.

Author: Do you use **XAMPP** or **MAMP** or **WampServer**? Most likely your answer is **no**.
That's why you raised this issue.
I use **XAMPP**. Therefore, after each change in file **php.ini**, **Apache** must be restarted.
I was involved in this for several hours. Finally decide to merge with you.

Reviewer 2: I don't use Windows.

Reviewer 2: In my understanding, PHP for Web (Apache) and PHP for CLI are different things.
If you run PHP via web, you need to restart Apache if you change the php.ini.
But PHP for CLI reads php.ini every time, so you don't need to do something.

> The configuration file (php.ini) is read when PHP starts up. For the server module versions of PHP, this happens only once when the web server is started. For the CGI and CLI versions, it happens on every invocation. 
https://www.php.net/manual/en/configuration.file.php

I don't see why XAMPP should restart Apache even with the CLI.

Reviewer: > I believe Apache has nothing to do with. `phpunit` runs on CLI.

Agree with this. You do not run tests on the web but on the CLI.
